src/Model/MyParcelConsignment.php

```diff
@@ -731,20 +931,23 @@ public function getDeliveryType()
         return $this->delivery_type;
     }
 
-	/**
-	 * The delivery type for the package
-	 *
-	 * Required: Yes if delivery_date has been specified
-	 *
-	 * @param int $delivery_type
-	 * @param bool $needDeliveryDate
-	 *
-	 * @return $this
-	 * @throws \Exception
-	 */
+    /**
+     * The delivery type for the package
+     *
+     * Required: Yes if delivery_date has been specified
+     *
+     * @param int $delivery_type
+     * @param bool $needDeliveryDate
+     *
+     * @return $this
+     * @throws \Exception
+     */
     public function setDeliveryType($delivery_type, $needDeliveryDate = true)
     {
-        if ($needDeliveryDate && $delivery_type !== 2 && $this->getDeliveryDate() == null) {
+        if ($needDeliveryDate &&
+            $delivery_type !== self::DELIVERY_TYPE_STANDARD &&
+            $this->getDeliveryDate() == null
```

Comment:
Ik weet niet hoe we dit elders doen, maar && aan het begin van de regel is misschien leesbaarderder